views/nodes/show.erb

```diff
@@ -3,25 +3,24 @@
     <a href="/node/<%= @node.id %>/edit">edit</a>
   </h2>
   <p>
-    Project Id: <%= @node.project_id %> <br>
```

Comment:
Make this the project name and remove the projects table below. A node only belongs to one project, and we don't need that project's details here - just a name that links back to the project view.